src/main/java/hwr/oop/List.java

```diff
@@ -34,12 +36,29 @@ public ToDoItem[] getListToDos() {
         return this.ListToDos;
     }
 
+    public String[] getBuckets() {
+        return Buckets;
+    }
+
+    public void setBuckets(String newbucket) {
+        String[] oldbucket = this.Buckets;
+        this.Buckets = new String[Buckets.length + 1];
```

Comment:
Das wird mit Listen simpler.

Reply:
siehe vor herigen Comment

Reply:
wurde an Listen angepasst und in addBucket umbenannt